### PR TITLE
Fix the selection of the surveyTable view for TSV export.

### PR DESCRIPTION
**Pull request by gijskant**

### CODE REVIEW

File: src/app/utilities/transmart-utilities/transmart-mapper.ts [hunk 1]
```diff
@@ -136,11 +136,7 @@ export class TransmartMapper {
             let el = new TransmartExportElement();
             el.dataType = dataType.name;
             el.format = fileFormat.name;
-            if (fileFormat.name === 'TSV' && dataType.name === 'clinical') {
-              el.dataView = 'dataTable';
-            } else {
-              el.dataView = defaultDataView;
-            }
+            el.dataView = defaultDataView;
```
Comment (beaugogh) [inline src/app/utilities/transmart-utilities/transmart-mapper.ts]: why the change here?

Comment (gijskant) [reply src/app/utilities/transmart-utilities/transmart-mapper.ts]: If the dataView is set to 'surveyTable', the TSV export would still choose 'dataTable' instead. The idea was that this setting determines what dataView is selected for the exports.
